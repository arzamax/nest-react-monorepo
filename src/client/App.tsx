import React from 'react';

export const App = () => <div>React App</div>;
