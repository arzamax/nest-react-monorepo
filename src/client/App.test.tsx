import React from 'react';
import { render } from '@testing-library/react';

import { App } from './App';

const setup = () => render(<App />);

describe('App test', () => {
  test('snapshot', () => {
    const { asFragment } = setup();

    expect(asFragment()).toMatchSnapshot();
  });

  test('content', () => {
    const { getByText } = setup();

    expect(getByText('React App')).not.toBeNull();
  });
});
