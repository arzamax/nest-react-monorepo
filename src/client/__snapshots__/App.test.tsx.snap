// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App test snapshot 1`] = `
<DocumentFragment>
  <div>
    React App
  </div>
</DocumentFragment>
`;
